javascriptv3/example_code/personalize/src/personalize_createBatchInferenceJob.js

```diff
@@ -0,0 +1,70 @@
+
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createBatchInferenceJob.js demonstrates how to create a batch inference job with Amazon Personalize.
+A batch inference job is a tool that imports your batch input data from an Amazon S3 bucket, uses your 
+solution version to generate recommendations, and exports the recommendations to an Amazon S3 bucket.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateBatchInferenceJob.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateBatchInferenceJob.html.